### ISSUE TITLE
add a method to project bipartite graphs

### ISSUE THREAD

**Issue by SNAnalyst** [SNAnalyst/SNAfun#25]
perhaps call this `to_unipartite`?
or, `make_projection`?

**Comment by SNAnalyst** [comment]
then also update the measures tutorial and potentially other tuts and slide decks as  well

**Comment by czucca** [comment]
I vote for 'make_projection'. Can you decide on which partition you project by using an argument?

**Comment by SNAnalyst** [comment]
yes, that would be an idea. Alternatively, the output can be a list with the two partitions. What would you suggest?

**Comment by czucca** [comment]
I think it would be good to have an argument that says "one", "two", or "both". By default, set to both outputs the list with the two partitions

**Comment by SNAnalyst** [comment]
added
